### PR TITLE
Update chaininfo and add to IBC

### PR DESCRIPTION
**Pull request by nodebreaker0-0**

### CODE REVIEW

File: crescent/assetlist.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
       "denom_units": [
         {
           "denom": "ubcre",
-          "exponent": 0,
+          "exponent": 6,
```
Comment (JeremyParish69) [inline crescent/assetlist.json:34]: This should remain at 0, as all base denoms do. bcre will have exponent 6, but ubcre should have 0.

File: _IBC/crescent-evmos.json [hunk 2]
```diff
@@ -2,22 +2,22 @@
     "$schema": "../ibc_data.schema.json",
     "chain_1": {
       "chain_name": "crescent",
-      "client_id": "07-tendermint-24",
-      "connection_id": "connection-27"
+      "client_id": "07-tendermint-12",
```
Comment (JeremyParish69) [inline _IBC/crescent-evmos.json:5]: why is this an edit to the crescent-stargaze file? Shouldn't this be a new file?

Comment (Heheehd) [reply _IBC/crescent-evmos.json:5]: Jejeje

**Comment by nodebreaker0-0** [comment]
Also, the crescent core version has been upgraded, and Terra2 has been newly added.

**Comment by JeremyParish69** [comment]
@nodebreaker0-0 Please restore the crescent-stargaze.json file this deletes.

**Comment by nodebreaker0-0** [comment]
@JeremyParish69 The ibc channel between Crescent and Stargaze is currently unmanaged and has been expired, which can cause confusion. And yet, do we have to restore it?

**Comment by JeremyParish69** [comment]
> @JeremyParish69 The ibc channel between Crescent and Stargaze is currently unmanaged and has been expired, which can cause confusion. And yet, do we have to restore it?

You don't have to restore the connection, but we shouldn't delete that file. If a new connection between the two chains is someday created, then the case for replacement can be examined.

**Comment by nodebreaker0-0** [comment]
@JeremyParish69 Restored Stargaze, added Juno. Thank you for your feedback.

**Pull request merged**